src/components/HeaderBar/HeaderBar.tsx

```diff
@@ -1,19 +1,24 @@
-import { Navbar, Container, ButtonGroup, Button } from 'react-bootstrap';
+import { Navbar, Container, ButtonGroup, Button, useAccordionButton } from 'react-bootstrap';
```

Comment:
It looks like `userAccordionButton` is never used?

Reply:
I did not add, it had been added previously on working tree I did not change it!